src/main/java/net/mcreator/ui/minecraft/villagers/JVillagerTradeProfession.java

```diff
@@ -54,12 +53,13 @@ public JVillagerTradeProfession(MCreator mcreator, IHelpContext gui, JPanel pare
 			List<JVillagerTradeProfession> professionList) {
 		super(mcreator, new BorderLayout(), gui);
 
+		this.workspace = mcreator.getWorkspace();
+
 		setOpaque(false);
 
+		villagerProfession = new DataListComboBox(mcreator, ElementUtil.loadAllVillagerProfessions(mcreator.getWorkspace()));
```

Comment:
I believe the `workspace` field can be used here instead.

Reply:
Done!